src/platform/packages/private/kbn-esql-editor/src/esql_editor.tsx

```diff
@@ -539,6 +540,7 @@ export const ESQLEditor = memo(function ESQLEditor({
         };
       },
       getInferenceEndpoints: kibana.services?.esql?.getInferenceEndpointsAutocomplete,
+      getLicense: () => getESQLLicense(kibana.services?.esql?.getLicense),
```

Comment:
nit: why do you define this `getESQLLicense` function in-between here, why not inline it here?